### ISSUE TITLE
Latest version is failing due to new terraform version apparently

### ISSUE THREAD

**Issue by adejanovski** [thelastpickle/tlp-cluster#110]
The terraform.tf.json file seems to be missing some informations that Terraform wants : 

```
2019-05-23 15:15:27,506 ERROR c.t.t.Docker$runContainer$4 [dockerjava-netty-1-3]   on terraform.tf.json line 93, in resource.aws_security_group.accelerate1_TlpClusterSG_1558624155:

2019-05-23 15:15:27,506 ERROR c.t.t.Docker$runContainer$4 [dockerjava-netty-1-3]   93:         "ingress" : [ {

2019-05-23 15:15:27,507 ERROR c.t.t.Docker$runContainer$4 [dockerjava-netty-1-3] 

2019-05-23 15:15:27,507 ERROR c.t.t.Docker$runContainer$4 [dockerjava-netty-1-3] Inappropriate value for attribute "ingress": element 0: attributes

2019-05-23 15:15:27,507 ERROR c.t.t.Docker$runContainer$4 [dockerjava-netty-1-3] "ipv6_cidr_blocks", "prefix_list_ids", and "security_groups" are required.


2019-05-23 15:15:27,508 ERROR c.t.t.Docker$runContainer$4 [dockerjava-netty-1-3] 

2019-05-23 15:15:27,508 ERROR c.t.t.Docker$runContainer$4 [dockerjava-netty-1-3] Error: Incorrect attribute value type


2019-05-23 15:15:27,509 ERROR c.t.t.Docker$runContainer$4 [dockerjava-netty-1-3]   on terraform.tf.json line 122, in resource.aws_security_group.accelerate1_TlpClusterSG_1558624155:

2019-05-23 15:15:27,509 ERROR c.t.t.Docker$runContainer$4 [dockerjava-netty-1-3]  122:         "egress" : [ {


2019-05-23 15:15:27,509 ERROR c.t.t.Docker$runContainer$4 [dockerjava-netty-1-3] Inappropriate value for attribute "egress": element 0: attributes

2019-05-23 15:15:27,510 ERROR c.t.t.Docker$runContainer$4 [dockerjava-netty-1-3] "ipv6_cidr_blocks", "prefix_list_ids", and "security_groups" are required.

2019-05-23 15:15:27,510 ERROR c.t.t.Docker$runContainer$4 [dockerjava-netty-1-3] 
```

**Comment by ossarga** [comment]
I've been able to reproduce this error. Terraform was recently updated to `0.12.0`. The attributes in the error message that are listed as required contradicts what the documentation says: https://www.terraform.io/docs/providers/aws/r/security_group.html. Despite this, I did add the required attributes to the _terraform.tf.json_ file and it made no difference. Terraform started complaining that other attributes which were already specified in the JSON file were missing.

@rustyrazorblade I'm going to peg the Terraform version at `0.11.14` until this issue gets resolved.
